README.md

```diff
@@ -177,9 +177,30 @@ and keys in shared memory, then set variable with the zone name.
 
 ## Automatic Certificate Renewal
 
-NGINX and NJS do not yet have a mechanism for running code on a time interval, which presents a challenge for certificate renewal. One workaround to this is to set something up to periodically request `/acme/auto` from the NGINX server.
+Certificate renewal is automatically managed through active healthchecks to send a `GET` request to `/acme/auto` every 90 seconds. The relevant configuration section is
 
-If running directly on a host, you can use `cron` to schedule a periodic request. When deploying in Kubernetes you can use a [liveness-check](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request). If you are running in a `docker` context, you can use Docker's `healthcheck:` functionality to do this.
+```nginx
+  # Internal upstream for automated certificates renewal
+  upstream acme_auto_renewal {
+    zone acme_auto_renewal 64k;
+    server 127.0.0.1:8000;
+  }
+
+  ## Internal certificates renewal server
+  # GETs 127.0.0.1:8000/acme/auto every 90 seconds to automatically renews certificates
+  server {
+    location /internal_auto_renewal {
+      internal;
+
+      health_check interval=90;
```

Comment:
Great idea @fabriziofiorucci ! When I try this on my server (nginx-1.25.1) I get this error:
```
web-proxy-nginx-1  | nginx: [emerg] unknown directive "health_check" in /etc/nginx/nginx.conf:49
```
Looks like this may only be a non-OSS feature?

Reply:
Active health checks are a NGINX Plus feature only and I found a bug in this, fixing now

Reply:
Targeting OSS users is an important part of this project. NGINX is losing mindshare/marketshare due to the relative difficulty of doing basic things like HTTPS and auto-cert-renewal in OSS.

Reply:
That makes sense, there could be a fully automated option for those who wanted to use NGINX Plus, while OSS users might rely on k8s/crontab.

Reply:
There are some NJS features nearing release that will let us run JS code on a time interval, so I think that solution will work for all :)

Reply:
I can offer to add a separate plus-specific file under `examples/nginx_plus.conf` and have a separate section about the differences between OSS and plus wrt this auto-acme configuration.

Reply:
Yeah IMO we should be OSS-first in our solution posture here.

Reply:
@ivanitskiy @zsteinkamp if that works for you I can amend this PR with a dedicated nginx_plus.conf file.

Reply:
it would be nice to separate OSS and Plus and don't mix them in a single example. (e.g. in Plus we can use KV instead of shared_dict and so on...)